pages/api-reference/connect/webhook-subscriptions/index.mdx

```diff
@@ -43,6 +43,35 @@ Webhooks are delivered in near real-time as events tied to your application are
 |      7       |               24 h                |                  48 h                   |
 |      8       |               24 h                |                  72 h                   |
 
+### List of Webhook Events
+
+The following list contains all supported events that can trigger a webhook notification to be sent using Dwolla Connect. New events may be added in the future, so be sure to check back on our [changelog](https://developers.dwolla.com/changelog) often. We also recommend subscribing to developer/product related updates to stay informed about the latest changes.
+
+#### Funding Sources
+
+| Event Topic Name                      | Description                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| funding_source:created                | **Description:** A funding source was added to a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Create a funding source for an account](https://developers.dwolla.com/api-reference/connect/funding-sources/https://developers.dwolla.com/api-reference/connect/funding-sources/create-for-account) endpoint, or when a funding source is added via a third party bank verification method. |
```

Comment:
```suggestion
| funding_source:created                | **Description:** A funding source was added to a main Dwolla Account.<br /> **Timing:** Occurs upon a POST request to the [Create a funding source for an account](/api-reference/connect/funding-sources/create-for-account) endpoint, or when a funding source is added via a third party bank verification method. |
```

Reply:
Good catch! @ShreyaThapa  Are you good with keeping these as absolute URLs vs. relative URLs? Absolute URLs are more reliable, SEO friendly, and work better with LLMs

Reply:
Ah, gotcha. That’s good insight! Then that sounds good to me!  

Thanks!